man/man1/oci-image-migrate.1

```diff
@@ -3,7 +3,6 @@
 .\"
 .\" Copyright (c) 2017, 2019 Oracle and/or its affiliates. All rights reserved.
 .\"
```

Comment:
Copyright message still not complete here

Reply:
fixed